Lecture-14/src/main/java/service/ApacheRepositoryService.java

```diff
@@ -0,0 +1,41 @@
+package service;
+
+import client.IRestClient;
+import constant.HttpMethodType;
+import deserializer.Deserializer;
+import model.GitHubResponse;
+import model.GitUser;
+
+import java.util.List;
+
+public class ApacheRepositoryService implements IRepositoryService {
+
+    private final IRestClient restClient;
+    private static final String URI_COLLABORATORS = "https://api.github.com/repos/mexal17/Lecture-2/collaborators";
```

Comment:
`https://api.github.com/` part should be constructed in `IRestClient`
If it's changed, e.g. `/v3/` added - you will have to change it in all services.

Reply:
done